README.md

```diff
@@ -53,19 +53,20 @@ github {
 
 where:
 
-  * `ip-address` is either an IPv4 or IPv6 address and `port` a port on which an
-    HTTP server will listen.
-  * `path` is the directory where the per-repo programs are stored. For a
-    repository `repo` owned by `user` the command:
+ * `ip-address` is either an IPv4 or IPv6 address and `port` a port on which an
+   HTTP server will listen.
+ * `cmd` is the command that will be executed when a webhook is received. In
+   this case, `/path/to/prps` is a path to a directory where per-repo programs
+   are stored. For a repository `repo` owned by `user` the command:
 
-      ```
-      <reposdir>/<user>/<repo> <event> <path to GitHub JSON>
-      ```
+     ```
+     /path/to/prps/<user>/<repo> <event> <path to GitHub JSON>
```

Comment:
We should probably use `<path-to-github-json>` so be consistent with other notations in this repo.

Reply:
Agreed. Fixed in https://github.com/softdevteam/snare/pull/43/commits/535a30bba7493feeef32ea47f6797bdbe829fc0b.